scripts/mbedtls_dev/psa_storage.py

```diff
@@ -101,25 +101,40 @@ class Key:
     LATEST_VERSION = 0
     """The latest version of the storage format."""
 
+    EXTENDABLE_USAGE_FLAGS = {
```

Comment:
`IMPLICIT_USAGE_FLAGS` would be a more descriptive name. The flags aren't extendable.